Library/Homebrew/formula_creator.rb

```diff
@@ -201,6 +201,18 @@ def install
             virtualenv_install_with_resources
         <% elsif @mode == :ruby %>
             ENV["GEM_HOME"] = libexec
+
+            # Install dependencies declared in a Gemfile.lock
+            # system "bundle", "config", "set", "without", "development", "test"
+            # system "bundle", "install"
+
+            # Install dependencies declared as resources
```

Comment:
How often is this the case for new Ruby formulae? Seems like a `Gemfile.lock` is pretty standard now?

Reply:
None in last 5 new Ruby formulae, but yes in last 10:
1. https://github.com/Homebrew/homebrew-core/pull/173274 - Gemfile.lock
2. https://github.com/Homebrew/homebrew-core/pull/172077 - via `gem install`
3. https://github.com/Homebrew/homebrew-core/pull/161231 - Gemfile.lock
4. https://github.com/Homebrew/homebrew-core/pull/157910 - Gemfile.lock
5. https://github.com/Homebrew/homebrew-core/pull/152307 - https://github.com/Homebrew/homebrew-core/pull/161343
6. https://github.com/Homebrew/homebrew-core/pull/136553 - resources, but dropped due to license
7. https://github.com/Homebrew/homebrew-core/pull/126915 - resources
8. https://github.com/Homebrew/homebrew-core/pull/126482 - via `gem install`

---

May also need to decide if `gem install` should be valid way of installing dependencies, or we should require `--ignore-dependencies`.

Reply:
> * [ronn-ng 0.10.1 (new formula) homebrew-core#172077](https://github.com/Homebrew/homebrew-core/pull/172077) - via `gem install`

This had a Gemfile.lock, I wonder why we didn't use it. Feels like it should be an audit failure if it has one and we don't use it.

`gem install` seems like it could be valid, then, if we need version pinning although personally I'd tend towards just saying we always use `bundle install` (given all of these have `Gemfile`s)